Data_Transfer_Hash/readme.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: pattern
+title: data transfer hash
+folder: Data_Transfer_Hash
+permalink: /patterns/Data_Transfer_Hash/
+categories: Architectural
+tags:
+ - Performance
+---
+
+## Also known as
+
+* data transfer hash
+
+## Intent
+
+Providing a  class called data transfer hash, in order to send message between other class.

```

Comment:
The intent should describe the pattern very clearly and concisely. Please try to rephrase.